app/Models/Feed.php

```diff
@@ -409,12 +409,19 @@ public function loadEntries($feed) {
 			$guid = $item->get_id(false, false);
 			$hasUniqueGuids &= empty($guids['_' . $guid]);
 			$guids['_' . $guid] = true;
+			$author_names = '';
+			if (is_array($authors)) {
+				foreach ($authors as $author) {
+					$author_names .= html_only_entity_decode(strip_tags($author->name == '' ? $author->email : $author->name)) . ', ';
```

Comment:
Will that work right in RTL scenarios?

Reply:
I have to admit that I have near-zero experience with RTL.
The current Hebrew translation seems to do something similar:

https://github.com/FreshRSS/FreshRSS/blob/32d9c3b7905f4e43ffdf4bf2bf37723cfd18390c/app/i18n/he/gen.php#L184

I guess it would also depend on the alphabet of authors' names, so I do not know how to make it better.

Reply:
Let's not worry about it prematurely then. :-)